tst/hooks/sdk/useMediaStreamMetrics.test.tsx

```diff
@@ -0,0 +1,191 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { act } from '@testing-library/react';
+import { renderHook } from '@testing-library/react-hooks';
+import {
+  AudioVideoObserver,
+  Backoff,
+  DefaultAudioVideoController,
+  DefaultAudioVideoFacade,
+  DefaultClientMetricReport,
+  DefaultReconnectController,
+  DefaultWebSocketAdapter,
+  MeetingSessionConfiguration,
+  MeetingSessionCredentials,
+  MeetingSessionURLs,
+  NoOpAudioVideoController,
+  NoOpDebugLogger,
+  NoOpMediaStreamBroker,
+} from 'amazon-chime-sdk-js';
+import React from 'react';
+
+import { AudioVideoContext } from '../../../src';
```

Comment:
Import from the `AudioVideoProvider`?

Reply:
The `AudioVideoContext` is exported in `index.ts` file, so it's better import it from `src` instead of `../src/../AudioVideoProvider. (As ESLint also recommended this).

e.g. if you are using React SDK, you will import from `amazon-chime-sdk-component-react`, instead of `amazon-chime-sdk-component-react/**/**/AudioVideoProvider`.

Reply:
So, should this be the case for all imports then?